jujugui/static/gui/src/app/components/deployment-flow/_deployment-flow.scss

```diff
@@ -23,27 +23,210 @@
         padding-top: 40px;
     }
 
+    &__section {
+        margin-bottom: 40px;
+        padding: 20px;
+        background-color: $light-grey;
+
+        &--clear {
+            padding: 0;
+            background-color: transparent;
+        }
+
+        &-title {
+            .svg-icon {
+                margin-right: 10px;
+                vertical-align: text-top;
+            }
+
+            &-aside {
+                padding-left: 20px;
+                font-size: 16px;
+            }
+        }
+    }
+
+    &__top-action {
+        float: right;
+    }
+
+    &__label {
+        margin-bottom: 10px;
+        color: $warm-grey;
+        font-size: 0.688em;
+    }
+
+    &__row-header {
+        font-weight: 400;
+    }
+
+    &__row-header,
+    &__row {
+        border-bottom: 1px solid $light-mid-grey;
+    }
+
+    .button--neutral {
```

Comment:
We should not reassign a core style as it can be confusing in the future. Could you add an additional class to this element which applies this modification.